erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -189,6 +191,12 @@ def on_submit(self):
 			from erpnext.accounts.doctype.pricing_rule.utils import update_coupon_code_count
 			update_coupon_code_count(self.coupon_code,'used')
 
+		flowkana_settings = frappe.get_doc("Flowkana Settings")
+
+		#send delivery request to flowkana if enabled by user
+		if flowkana_settings.enable_flowkana and self.fulfillment_partner == "Flowkana":
+			self.send_delivery_request_to_flowkana()
+
```

Comment:
Use frappe.hooks to integrate it with Flowkana, make sure all the flowkana code is in flowkana's controller